ods-document-generation-svc/openshift/is.yml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Template
+labels:
+  template: 'ods-document-generation-service'
+  app: 'ods-doc-gen-svc'

```

Comment:
Not sure if elsewhere we have labels on the template resource itself. If so, we can just remove them, as we process the template locally, it does not need labels.